### ISSUE TITLE
ionic-title not updating document title with async pipe

### ISSUE THREAD

**Issue by FdezRomero** [ionic-team/ionic#9124]
#### Short description of the problem:
When using an async pipe inside an `ion-title` the document title doesn't update as well. It seems the function to set the document title is only called once. This happens specially when using a Deeplinker fragment URL (e.g. `/#/home`).

#### What behavior are you expecting?
The document title to show the same resolved value as in the app navbar.

**Steps to reproduce:**
1. Pick a page registered in the Deeplinker and put an async pipe inside `ion-title`, for example `<ion-title>{{ 'PAGE_TITLE' | translate }}</ion-title>`.
2. Reload the page.
3. The document title will show `PAGE_TITLE` instead of the resolved value.

**Which Ionic Version?** 1.x or 2.x
2.0.0-rc.2

#### Plunker that shows an example of your issue
The issue wouldn't show in a Plunker as it doesn't show the app's document title.

**Run `ionic info` from terminal/cmd prompt:** (paste output below)
Cordova CLI: 6.4.0
Ionic Framework Version: 2.0.0-rc.2
Ionic CLI Version: 2.1.8
Ionic App Lib Version: 2.1.4
Ionic App Scripts Version: 0.0.43
ios-deploy version: 1.9.0
ios-sim version: 5.0.11
OS: macOS Sierra
Node Version: v6.9.1
Xcode version: Xcode 8.1 Build version 8B62

**Comment by mebibou** [comment]
Have the same problem, but in my case the translation does change the title, but this situation doesn't:
```html
<ion-title>{{configuration ? configuration.name : ('loading' | translate)}}</ion-title>
```

`configuration` is fetched via an api, when I do get the `configuration` the title within the header of the page changes but not the chrome page title (which always says "Loading"). This is not a problem for an app but it is for a website

**Comment by mebibou** [comment]
@jgw96 fyi, the issue has partially been fixed in latest versions (like `3.7.1`), so the title of the tab gets updated after translation arrives, but when navigating back from a page to another, the title does not get updated. See issue #9398

**Comment by kensodemann** [comment]
I am going to close this since there will not be another v3 release and this code should be fixed with the next major release of Ionic and Ionic Angular. If this is still an issue after that release is out please let us know.
